lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -35,49 +35,67 @@ class SynchronizeLdm < BaseAction
 
         description 'Allows to have facts with higher precision decimals'
         param :exclude_fact_rule, instance_of(Type::BooleanType), required: false, default: false
+
+        description 'Specifies how to synchronize LDM and resolve possible conflicts'
+        param :synchronize_ldm, instance_of(Type::SynchronizeLDM), required: false, default: 'diff_against_master_with_fallback'
       end
 
       class << self
         def call(params)
           include_ca = params.include_computed_attributes.to_b
           exclude_fact_rule = params.exclude_fact_rule.to_b
-
-          results = []
-
           client = params.gdc_gd_client
-          development_client = params.development_client
+          synchronize = []
+          results = []
+          synchronize = params.synchronize.map do |segment_info|
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)